INFO - 2024-03-19 10:25:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-03-19 10:38:23 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\MySQLi\Connection.php(173): mysqli->real_connect()
#1 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\BaseModel.php(618): CodeIgniter\Model->doFindAll()
#6 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\app\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#9 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Commands\Server\rewrite.php(47): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#3 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\BaseModel.php(618): CodeIgniter\Model->doFindAll()
#5 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\app\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#8 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#9 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Commands\Server\rewrite.php(47): require_once('...')
#11 {main}
CRITICAL - 2024-03-19 10:38:23 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll()
 5 APPPATH\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4-capstone101\\ci4_capstone1.0\\public\\index.php')
ERROR - 2024-03-19 10:38:27 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\MySQLi\Connection.php(173): mysqli->real_connect()
#1 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\BaseModel.php(618): CodeIgniter\Model->doFindAll()
#6 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\app\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#9 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#10 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Commands\Server\rewrite.php(47): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
#3 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\BaseModel.php(618): CodeIgniter\Model->doFindAll()
#5 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\app\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
#8 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
#9 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ci4-capstone101\ci4_capstone1.0\system\Commands\Server\rewrite.php(47): require_once('...')
#11 {main}
CRITICAL - 2024-03-19 10:38:27 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll()
 5 APPPATH\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4-capstone101\\ci4_capstone1.0\\public\\index.php')
CRITICAL - 2024-03-19 10:40:12 --> syntax error, unexpected token "<"
in APPPATH\Controllers\OpenAIController.php on line 11.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile()
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace()
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass()
 4 SYSTEMPATH\CodeIgniter.php(892): class_exists()
 5 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4-capstone101\\ci4_capstone1.0\\public\\index.php')
CRITICAL - 2024-03-19 10:43:31 --> syntax error, unexpected token "namespace"
in APPPATH\Controllers\OpenAIController.php on line 12.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile()
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace()
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass()
 4 SYSTEMPATH\CodeIgniter.php(892): class_exists()
 5 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4-capstone101\\ci4_capstone1.0\\public\\index.php')
CRITICAL - 2024-03-19 10:44:21 --> Incorrect API key provided: sk-n3GHi***************************************uu5O. You can find your API key at https://platform.openai.com/account/api-keys.
in VENDORPATH\openai-php\client\src\Transporters\HttpTransporter.php on line 131.
 1 VENDORPATH\openai-php\client\src\Transporters\HttpTransporter.php(57): OpenAI\Transporters\HttpTransporter->throwIfJsonError()
 2 VENDORPATH\openai-php\client\src\Resources\Completions.php(33): OpenAI\Transporters\HttpTransporter->requestObject()
 3 APPPATH\Libraries\GeneratorOpenAIService.php(18): OpenAI\Resources\Completions->create()
 4 APPPATH\Controllers\OpenAIController.php(25): App\Libraries\GeneratorOpenAIService->generateResponseOpenAi()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\OpenAIController->chatOpenAi()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4-capstone101\\ci4_capstone1.0\\public\\index.php')
CRITICAL - 2024-03-19 10:46:08 --> The model `text-davinci-003` has been deprecated, learn more here: https://platform.openai.com/docs/deprecations
in VENDORPATH\openai-php\client\src\Transporters\HttpTransporter.php on line 131.
 1 VENDORPATH\openai-php\client\src\Transporters\HttpTransporter.php(57): OpenAI\Transporters\HttpTransporter->throwIfJsonError()
 2 VENDORPATH\openai-php\client\src\Resources\Completions.php(33): OpenAI\Transporters\HttpTransporter->requestObject()
 3 APPPATH\Libraries\GeneratorOpenAIService.php(18): OpenAI\Resources\Completions->create()
 4 APPPATH\Controllers\OpenAIController.php(25): App\Libraries\GeneratorOpenAIService->generateResponseOpenAi()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\OpenAIController->chatOpenAi()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4-capstone101\\ci4_capstone1.0\\public\\index.php')
CRITICAL - 2024-03-19 10:48:16 --> You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.
in VENDORPATH\openai-php\client\src\Transporters\HttpTransporter.php on line 131.
 1 VENDORPATH\openai-php\client\src\Transporters\HttpTransporter.php(57): OpenAI\Transporters\HttpTransporter->throwIfJsonError()
 2 VENDORPATH\openai-php\client\src\Resources\Completions.php(33): OpenAI\Transporters\HttpTransporter->requestObject()
 3 APPPATH\Libraries\GeneratorOpenAIService.php(18): OpenAI\Resources\Completions->create()
 4 APPPATH\Controllers\OpenAIController.php(25): App\Libraries\GeneratorOpenAIService->generateResponseOpenAi()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\OpenAIController->chatOpenAi()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest()
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4-capstone101\\ci4_capstone1.0\\public\\index.php')
